assets/common/src/register.py

```diff
@@ -27,6 +28,9 @@
 
 SUPPORTED_MODEL_ASSET_TYPES = [AssetTypes.CUSTOM_MODEL, AssetTypes.MLFLOW_MODEL]
 MLFLOW_MODEL_FOLDER = "mlflow_model_folder"
```

Comment:
Is this the mandatory name for the folder? If yes, how are we communicating this to the customers?

Reply:
yes, for mlflow model, it has to be `mlflow_model_folder`. We are making sure to copy model files to `mlflow_model_folder` for mlflow model. MLmodel file has to exists under `mlflow_model_folder` or it will fail. For custom model we are publishing as it is